test/infrastructure/container/docker.go

```diff
@@ -430,15 +424,6 @@ func (d *dockerRuntime) RunContainer(ctx context.Context, runConfig *RunContaine
 	}
 	containerConfig.Env = envVars
 
-	// handle Docker on Btrfs or ZFS
-	// https://github.com/kubernetes-sigs/kind/issues/1416#issuecomment-606514724
-	if d.mountDevMapper(info) {
-		runConfig.Mounts = append(runConfig.Mounts, Mount{
-			Source: "/dev/mapper",
-			Target: "/dev/mapper",
-		})
-	}
```

Comment:
@cannonpalms could you remove this block and the associated code (`mountDevMapper`). Removing this alone makes this work for me when running btrfs.

Reply:
I didn't understand clearly what you meant here, so I took it to mean that you were asking me to remove the now-unused `mountDevMapper(...)` and `needsDevMapper(...)` functions, _not_ that this highlighted block alone was the only necessary change & I should restore the previous removal of the block consuming `needsDevMapper(...)`.

Is that correct?